eth/tracers/api.go

```diff
@@ -705,7 +795,9 @@ func (api *API) standardTraceBlockToFile(ctx context.Context, block *types.Block
 			canon = false
 		}
 	}
-	for i, tx := range block.Transactions() {
+
+	txs, stateSyncPresent := api.getAllBlockTransactions(ctx, block)
```

Comment:
As discussed, we don't want to change the current behavior, which won't include state sync transaction. Could we introduce a new flag to trace API and only include state sync transactions when the flag is set to true?

Reply:
Done